dashboard/app/controllers/foorm_preview_controller.rb

```diff
@@ -1,13 +1,19 @@
 class FoormPreviewController < ApplicationController
   # GET '/foorm/preview/:name'
   def index
-    render_404 if Rails.env.production?
+    if Rails.env.production?
```

Comment:
Can this be done in 1 line?
```suggestion
return render_404 if Rails.env.production?
```